lib/streamlit/elements/lib/built_in_chart_utils.py

```diff
@@ -140,6 +140,8 @@ def generate_chart(
     height: int | None = None,
     # Bar & Area charts only:
     stack: bool | ChartStackType | None = None,
+    sort_by: str | None = None,
+    order: Literal["ascending", "descending"] = "ascending",
```

Comment:
Does `order` have any usage outside of the usage with `sort_by`? If not, it feels a bit cleaner and less confusing to just keep this within a single parameter, e.g.: 

```
sort_by: str | Tuple[str, Literal["ascending", "descending"]] None = None,

-> sort_by = ("col_1", "descending")
-> sort_by = "col_1"
```

Reply:
Or we could simply allow `sort_by="-column1"` for descending, which is also supported in Altair.